runtime/queries/nix/locals.scm

```diff
@@ -0,0 +1,33 @@
+;; when using @local.reference, tree-sitter seems to 
```

Comment:
These can be removed. tree-sitter has difficulty detecting locals in Nix because they're lazy in Nix and always top-to-bottom in most other languages. (See the linked issue in the comment.)

I doubt tree-sitter will get support for locals tracking that would work for Nix and if it does I expect we would need to start from scratch with the queries.